teamMaster.py

```diff
@@ -0,0 +1,264 @@
+"""
+teamMaster.py
+Author: Robert Becker
+Date: May 21, 2017
+Purpose: use daily schedule to extract team data from gameday boxscore
+
+Setup file/url names based on optional date arugment (mm-dd-yyyy), or today
+Load daily schedule dictionary into memory
+Use schedule to search thru boxscore dictionaries to get all team info
+Update teamMaster dictionary with any new team data found
+"""
+
+
+import sys
+import shutil
+import argparse
+import logging
+import logging.config
+import datetime
+import json
+import requests
+
+
+# setup global variables
+SCRIPT = 'teamMaster.py'
```

Comment:
I agree that having the name coded in a variable is not ideal, and when I run the seasonPlayer module to run all three of the other modules in a bundle, that __name__ worked great. However when I run one of the scripts as a standalone function all I see is __main__ throughout the log. I don't find this useful as I'd like to know which script or function is production each message. Even when I run it as a bundle, the messages produced by seasonPlayer.py also say __main__ so when scanning through a busy log it wouldn't be clear to someone which script produced those particular messages. I tried to find something that would give me the script name but couldn't. Looking again, I just found this:

os.path.basename(__file__)

That gives me the name of the script's filename currently running, with the .py suffix. This works regardless if I run it as a function or as a standalone process. Acceptable to use this instead?